atomic_reactor/util.py

```diff
@@ -75,7 +77,9 @@ def to_str(self, registry=True, tag=True, explicit_tag=False,
 
         result = self.repo
 
-        if tag and self.tag:
+        if tag and self.tag and ':' in self.tag:
```

Comment:
Turns out that using `self.sep` is a little awkward because it must be computed in `__init__` as well as `parse` methods. Switched to using your initial suggestion.